temporallib/encryption/data_converter.py

```diff
@@ -10,8 +10,8 @@
 from pydantic_settings import BaseSettings
 
 class EncryptionOptions(BaseSettings):
-    key: str
-    compress: bool = False
+    key: Optional[str] = None
```

Comment:
so it was not enough the fact that the encryption is optional in the upper level?
there is no point in having encryption options when there is no key, no?

Reply:
No, because when you want to initialize `EncryptionOptions` using environment variables as follows:

```python
client = await Client.connect(
        client_opt=Options(auth=AuthOptions(config=GoogleAuthOptions()), encryption=EncryptionOptions()),)
```

it complains about the required `key` parameter.